python/lsst/pipe/supertask/activator.py

```diff
@@ -292,7 +321,11 @@ def load_super_task(super_taskname):
 
         if super_task_module:
             # Need to add a try -- except here
```

Comment:
Removed the line, this is from before I change few other things in the task/module discovery